google-cloud-spanner/src/main/resources/com/google/cloud/spanner/spi/v1/grpc-gcp-apiconfig.json

```diff
@@ -0,0 +1,106 @@
+{
+  "channelPool": {
+    "maxSize": 3,
+    "maxConcurrentStreamsLowWatermark": 1
+  },
+  "method": [
+    {
+      "name": ["google.spanner.v1.Spanner/CreateSession"],
+      "affinity" : {
+        "command": "BIND",
+        "affinityKey": "name"
+      }
+    },
+    {
+      "name": ["google.spanner.v1.Spanner/BatchCreateSessions"],
```

Comment:
This will (if I understand it correctly) bind the channel that is used to call `BatchCreateSessions` to the sessions that are returned by call. Is that correct?
Does this function use the same `CHANNEL_HINT` [option](https://github.com/googleapis/java-spanner/blob/2023839cce80de0ff6451a4b6274f5da9b18416f/google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpc.java#L81) as Gax?

The session pool in the Spanner client is initialized when a database client is created. By default it is filled with 100 sessions. These sessions are distributed over the available channels in the following way:
1. Divide the number of sessions that is needed by the number of available channels (i.e. 100 sessions / 4 channels = 25 sessions per channel)
2. [Iterate over each channel number](https://github.com/googleapis/java-spanner/blob/2023839cce80de0ff6451a4b6274f5da9b18416f/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionClient.java#L252) and execute `BatchCreateSessions(sessionCount=25)` for each channel. [Set the channel hint](https://github.com/googleapis/java-spanner/blob/2023839cce80de0ff6451a4b6274f5da9b18416f/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionClient.java#L288) in the request to the current channel number.

The above approach assumes that:
1. The available channels are handed out round-robin from the underlying channel pool.
2. Executing `BatchCreateSessions` on a specific channel and setting the channel hint will affiliate all the sessions that are returned by the invocation with the channel that was used for the invocation.
3. Subsequent RPCs that reference the affiliated session will automatically cause the RPC to be executed on the affiliated channel.

Will the above also work with the new channel pool?

Reply:
I'll resolve this one as it is the duplicate of above.

Reply:
Yeah, sorry. GitHub had some issues yesterday, and it appears that it added my comment twice.